Rondo/UI/Variables/VariablesViewController.swift

```diff
@@ -66,7 +68,81 @@ class VariablesViewController: FormViewController {
                 self.build()
             })
         })
+        
+        form +++ section
+        
+        buildDashbordVarGroupSection()
+    }
+    
+    func buildDashbordVarGroupSection () {
+        // Example for Variables Group defined on the Dashboard
+        // weapons (group)
+        // -> daggers (group)
+        // -> -> magic_dagger (group)
+        // -> -> -> name
+        // -> -> -> damage
+        // -> -> -> level
+        // -> -> -> crit
+        // -> -> -> icon (image file)
+        let name = weapons.object(forKeyPathComponents: ["daggers", "magic_dagger", "name"]) as? String
+        let damage = weapons.object(forKeyPathComponents: ["daggers", "magic_dagger", "damage"]) as? NSNumber
+        let level = weapons.object(forKeyPathComponents: ["daggers", "magic_dagger", "level"]) as? NSNumber
+        let crit = weapons.object(forKeyPathComponents: ["daggers", "magic_dagger", "crit"]) as? String
+        let icon = weapons.object(forKeyPathComponents: ["daggers", "magic_dagger", "icon"]) as? String
+        
+        var weaponImage:UIImage?
+        
+        if let fileName = icon {
+            // try to download the image
+            // if the image is already downloaded it will not trigger the onComplete callback
+            let willDownload = LPFileManager.maybeDownloadFile(fileName, defaultValue: "", onComplete: {
 
+                // Get the row from the form
+                let imageRow = self.form.allRows.last as? ImageRow
+                
+                weaponImage = self.getImageFile(fileName)
+                if let imgr = imageRow {
+                    // Change the image and update the cell
+                    imgr.value = weaponImage
+                    imgr.updateCell()
+                }
+            })
+
+            if !willDownload {
+                weaponImage = self.getImageFile(fileName)
+            }
+        }
+        
+        let section = Section(name)
+        section <<< LabelRow {
+            $0.title = "Level"
+            $0.value = level?.description
+        }
+        section <<< LabelRow {
+            $0.title = "Damage"
+            $0.value = damage?.description
+        }
+        section <<< LabelRow {
+            $0.title = "Critical Chance"
+            $0.value = crit?.description
+        }
+        
+        section <<< ImageRow {
+            $0.value = weaponImage
+        }.cellSetup { (cell, row) in
+            //  Change the cell's height
+            cell.height = { return CGFloat(200) }
+        }
+        
         form +++ section
     }
+    
+    func getImageFile(_ fileName: String) -> UIImage? {
+        let fl = LPFileManager.fileValue(fileName, withDefaultValue: fileName)
```

Comment:
Why not:
`if let file = LPFileManager.fileValue(fileName, withDefaultValue: fileName)`
There is no need of creating local let fl